src/modules/previewpane/powerpreview/trace.cpp

```diff
@@ -73,59 +45,18 @@ void Trace::PowerPreviewSettingsUpDateFailed(LPCWSTR SettingsName)
 {
     TraceLoggingWrite(
         g_hProvider,
-        "PowerPreview_FilePreview_FailedUpdatingSettings",
+        "PowerPreview_TweakUISettings_FailedUpdatingSettings",
         TraceLoggingWideString(SettingsName, "ExceptionMessage"),
         ProjectTelemetryPrivacyDataTag(ProjectTelemetryTag_ProductAndServicePerformance),
         TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
         TraceLoggingKeyword(PROJECT_KEYWORD_MEASURE));
 }
 
-void Trace::PreviewPaneSVGRenderEnabled()
```

Comment:
Will we able to get the telemetry of which specific preview handlers are enabled if this code is removed?

Reply:
Yes it is done in line 201 and 214 of the settings.cpp file https://github.com/microsoft/PowerToys/blob/user/lamotile/imporve_integration/src/modules/previewpane/powerpreview/settings.cpp